lib/index.js

```diff
@@ -73,15 +75,20 @@ class Client {
     return response.data
   }
 
-  put(actionType, data) {
-    return ActionApi.action(this.api, this.apiKey, actionType, data)
+  push(actionType, payload) {
+    const file = f11s.open(payload)
+    const dataset = new f11s.Dataset({
+      ...file.descriptor,
+      resources: [],
+    })
+    return this.action(actionType, dataset.descriptor)
   }
 
   /**
    * @param {File} file
```

Comment:
I think you should update this JSDoc. Also we talked about this, just want to confirm, file is a browser `File`? I remember we agreed to have `file` as `datajs` file, so all the funcitons from datajs will be available, but now I'm thinking maybe we should get anything like we get as in case of `datajs.open()` and convert it to datajs file inside `pushBlob()` function? In other words, the question is, should we expect from the users to pass `datajs` like file or anything which is convertible `datajs` so we will convert ourselves? @rufuspollock please let us know your thoughts